features/telemetry/common/src/main/java/org/opennms/netmgt/telemetry/common/ipc/TelemetrySinkModule.java

```diff
@@ -120,17 +120,23 @@ public int getCompletionIntervalMs() {
 
             @Override
             public Object key(TelemetryMessage telemetryMessage) {
-                return telemetryMessage.getSource();
```

Comment:
Can we set a fake source rather than keeping it null? I'm thinking `host=0.0.0.0` and `port=0`

Reply:
Done.